### PR TITLE
add \CHttpSession::setCacheLimiter

### PR DESCRIPTION
**Pull request by b1rdex**
(cherry picked from commit b6306908cfcd59d84269739ea3bd845de78c8be1)

### CODE REVIEW

**Comment by b1rdex** [comment]
Done.

**Comment by samdark** [comment]
Merged. Thank you!

File: framework/web/filters/CHttpCacheFilter.php [hunk 1]
```diff
@@ -193,7 +193,7 @@ protected function sendCacheControlHeader()
 	{
 		if(Yii::app()->session->isStarted)
 		{
-			session_cache_limiter('public');
+			Yii::app()->session->setCacheLimiter('public');
```
Comment (rob006) [inline framework/web/filters/CHttpCacheFilter.php]: What is purpose of `session_cache_limiter()` here? This change broken `CHttpCacheFilter` in my case - for some reason I get `last-modified` header always set to mtime of entry script (which is obviously incorrect), and I'm unable to get 304 response from this filter.

Comment (samdark) [reply framework/web/filters/CHttpCacheFilter.php]: These should be equal.

Comment (rob006) [reply framework/web/filters/CHttpCacheFilter.php]: Disabling and reenabling session is definitely not transparent - I was also wondering why each requests sets session cookie, and this was the reason :P.

**Pull request merged**